modules/ROOT/pages/inline-css.adoc

```diff
@@ -0,0 +1,133 @@
+= Inline CSS plugin
+:navtitle: Inline CSS
+:description: takes {productname} documents and outputs HTML formatted according to email-friendly HTML best practices (eg inlined CSS).
+:description_short: Inlines {productname} CSS for better presentation in email.
+:keywords: plugin, inlinecss, email friendly, email, output, inline, css, post-process, html
+:pluginname: Inline CSS
+:plugincode: inlinecss
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+As is normal and long-time best practice, {productname} documents keep CSS and HTML separate. This practice is not, however, email-friendly.
+
+The {pluginname} plugin takes a {productname} document, and processes it such that the previously separate CSS is applied inline to each HTML element. This single output file is much closer to the common requirements for sending as an HTML-formatted email.
+
+== Interactive example
+
+liveDemo::inline-css[]
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'inlinecss',
+});
+----
+
+== Usage notes
+
+The {pluginname} plugin does not support
+
+. https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes[Pseudo-classes] (for example `button:hover { color: blue; }`).
+. https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements[Pseudo-elements] (for example `p::first-line { color: blue; }`)
+. https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule[@ rules] (for example `@media (screen)`)
+. The `!important` property.
+
+The {pluginname} plugin does support
+
+. At-rules other than @import.
```

Comment:
I probably was not clear. The only At-Rule that is supported is `@import`. All other At-rules are not supported for 6.3.1

Reply:
@Skylite73 The mention of . At-rules other than @import. has been removed at this stage.
@brian-forte ref only.